homeassistant/components/harman_kardon_avr/media_player.py

```diff
@@ -35,42 +56,59 @@ def setup_platform(hass, config, add_entities, discover_info=None):
     name = config[CONF_NAME]
     host = config[CONF_HOST]
     port = config[CONF_PORT]
+    key_interval = config[CONF_KEY_INTERVAL]
```

Comment:
I don't see a use for this either, and it's only used for the fake volume? We should drop it.